### PR TITLE
Feature/external legislation

### PR DESCRIPTION
**Pull request by kowal**
Added `ExternalLegislation` resource + CRUD

Zrzut ekranu 2019-08-23 o 12 34 48
Zrzut ekranu 2019-08-23 o 12 34 56

### CODE REVIEW

File: db/migrate/20190822112921_create_external_legislations.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateExternalLegislations < ActiveRecord::Migration[5.2]
+  def change
+    create_table :external_legislations do |t|
+      t.string :name
```
Comment (tsubik) [inline db/migrate/20190822112921_create_external_legislations.rb]: we could add not null here

**Comment by kowal** [comment]
> 👍 Just one small change to add not null and validation.
> 
> Adding external legislation to litigation form is a separate task? I don't remember. We could use selectedList component the same as for normal legislations.

yes, it will be separate task/PR

File: db/migrate/20190822112921_create_external_legislations.rb (hunk 1 above)
Comment (kowal) [reply db/migrate/20190822112921_create_external_legislations.rb]: updaded

File: app/admin/documents.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 ActiveAdmin.register Document do
+  config.batch_actions = false
```
Comment (kowal) [inline app/admin/documents.rb]: I moved `config.` statements to the top everywhere

**Comment by tsubik** [comment]
Ahh ok, that was as another task. I will merge this one then.

**Pull request merged**